gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/template_node_driver.py

```diff
@@ -135,6 +135,9 @@ def validate_create(self, context):
 
     @log.log
     def validate_update(self, context):
+        if context.original_profile is None:
+            return True
```

Comment:
I think this needs to be  updated w.r.t latest heat driver. return is not requied